specs/pvs/corla.pvs

```diff
@@ -1053,9 +1136,9 @@ BEGIN
 %%% ballot polling audits, but in no event will the risk limit exceed
```

Comment:
Yes it is, which is why I have included the text.  We can/should make it clear what parts of the specification are actually realized in the system we are building, but that comes as a part of the description of the refinement, which happens as we write the code this week.